NitroxModel/DataStructures/GameLogic/Entities/Metadata/EntityMetadata.cs

```diff
@@ -24,14 +24,14 @@ namespace NitroxModel.DataStructures.GameLogic.Entities.Metadata
     [ProtoInclude(190, typeof(CrafterMetadata))]
     [ProtoInclude(200, typeof(PlantableMetadata))]
     [ProtoInclude(210, typeof(CyclopsMetadata))]
-    [ProtoInclude(220, typeof(SeamothMetadata))]
-    [ProtoInclude(230, typeof(SubNameInputMetadata))]
-    [ProtoInclude(240, typeof(RocketMetadata))]
-    [ProtoInclude(250, typeof(CyclopsLightingMetadata))]
-    [ProtoInclude(260, typeof(FireExtinguisherHolderMetadata))]
-    [ProtoInclude(270, typeof(PlayerMetadata))]
-    [ProtoInclude(280, typeof(GhostMetadata))]
-    [ProtoInclude(290, typeof(WaterParkCreatureMetadata))]
+    [ProtoInclude(220, typeof(SubNameInputMetadata))]
+    [ProtoInclude(230, typeof(RocketMetadata))]
+    [ProtoInclude(240, typeof(CyclopsLightingMetadata))]
+    [ProtoInclude(250, typeof(FireExtinguisherHolderMetadata))]
+    [ProtoInclude(260, typeof(PlayerMetadata))]
+    [ProtoInclude(270, typeof(GhostMetadata))]
+    [ProtoInclude(280, typeof(WaterParkCreatureMetadata))]
+    [ProtoInclude(290, typeof(NamedColoredMetadata))]
```

Comment:
Change numbers to 50+n cf https://github.com/SubnauticaNitrox/Nitrox/pull/2087#discussion_r1409719990